public/iter.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
```

Comment:
This file `public/iter.html` should also be deleted and not committed. To delete you have to erase the file and then `git add .` -> `git commit` ...etc.